lib/matplotlib/contour.py

```diff
@@ -1333,7 +1305,7 @@ def find_nearest_contour(self, x, y, indices=None, pixel=True):
         xmin, ymin : float
             The point in the contour plot that is closest to ``(x, y)``.
         d : float
-            The distance from ``(xmin, ymin)`` to ``(x, y)``.
+            The squared distance from ``(xmin, ymin)`` to ``(x, y)``.
```

Comment:
This seems to be public, so I'm not sure we should change this.

Reply:
I'm just describing the actual behavior, not changing it.